config/config.go

```diff
@@ -36,8 +44,9 @@ type Server struct {
 // Config defines full YAML configuration.
 // USED FOR YAML
 type Config struct {
-	Identity security.Identity ",omitempty"
-	Server   Server            ",omitempty"
+	Identity 		  security.Identity ",omitempty"
+	Server   		  Server            ",omitempty"
+	PrometheusService string 			",omitempty"
```

Comment:
the way the Go YAML processing/parsing works, the name of the yaml property is all lowercase so this is going to be a running word "prometheusservice" in the yaml file or configmap. You should add an underscore _ between words as just having camel-case isn't enough. If this is named Prometheus_Service, the yaml processor will look for the property named "promehteus_service" which is what we would want to make it easier to read.

Reply:
Just like the env var, we should append _Url to this to make it clear its a url (`Prometheus_Service_Url`)

Reply:
+1